test/ut3_user/expectations/test_expectation_anydata.pkb

```diff
@@ -957,6 +957,23 @@ Rows: [ 60 differences, showing first 20 ]
     ut.expect(ut3_tester_helper.main_helper.get_failed_expectations_num).to_equal(0); 
   end;  
   
+   procedure collection_not_to_contain is
+    l_actual           ut3_tester_helper.test_dummy_object_list;
+    l_expected         ut3_tester_helper.test_dummy_object_list;
+  begin
+    --Arrange
+    select ut3_tester_helper.test_dummy_object( rownum, 'Something2 '||rownum, rownum+100)
+      bulk collect into l_actual
+      from dual connect by level <=4;
+    select ut3_tester_helper.test_dummy_object( rownum, 'Something '||rownum, rownum)
+      bulk collect into l_expected
+      from dual connect by level <=2
+     order by rownum desc;
+    --Act
+    ut3_develop.ut.expect(anydata.convertCollection(l_actual)).not_to_contain(anydata.convertCollection(l_expected));
```

Comment:
shouldn't we be testing for `not_to( contain() )` in this PR mainly?
I cant understand why this test was added here.

Reply:
Mainly for achieving code coverage quality gate.
We were changing the line of code I expectation that was referring to anydata not contain and line was coming as not covered.

Reply:
@jgebal  Is that make sense or you want me to remove that test but then it will complain on the code coverage :)